src/encode.rs

```diff
@@ -21,12 +21,13 @@ const STRICT_INT_MAX: i64 = 9007199254740991;
 
 pub const RECURSION_LIMIT: u8 = 255;
 
-pub const NON_STR_KEYS: u16 = 1 << 8;
+pub const STRICT_INTEGER: u16 = 1;
 pub const SERIALIZE_DATACLASS: u16 = 1 << 4;
-pub const SERIALIZE_NUMPY: u16 = 1 << 7;
 pub const SERIALIZE_UUID: u16 = 1 << 5;
 pub const SORT_KEYS: u16 = 1 << 6;
-pub const STRICT_INTEGER: u16 = 1;
+pub const SERIALIZE_NUMPY: u16 = 1 << 7;
+pub const NON_STR_KEYS: u16 = 1 << 8;
+pub const PRETTY: u16 = 1 << 11;
```

Comment:
not sure if there's some specific logic behind these bit shifts. I reordered them in order of their offset and added this new one as 11

Reply:
This introduces the bugs that 1 << 10 is recognized as an option but isn't and 1 << 9 is used internally but the user is allowed to overwrite it.